pkg/asset/installconfig/gcp/session.go

```diff
@@ -78,40 +79,70 @@ func getCredentials(ctx context.Context) (*googleoauth.Credentials, error) {
 	return creds, nil
 }
 
+func lazyInit() {
+	if len(credLoaders) > 0 {
+		return
+	}
+
+	for _, authEnv := range authEnvs {
+		credLoaders = append(credLoaders, &envLoader{env: authEnv})
+	}
+	credLoaders = append(credLoaders, &fileLoader{path: defaultAuthFilePath})
+	credLoaders = append(credLoaders, &cliLoader{})
```

Comment:
also why are we moving the cred loaders to init from loadCredentials?

Reply:
`onceLoggers` must be created only once in the lifetime of the process, and it depends on the cred loaders being known too. But as mentioned in the comment above, I removed the function and made that logic part of `loadCredentials`, only with the difference of being lazy initialized.